app/src/main/java/nu/yona/app/ui/frinends/TimelineStickyAdapter.java

```diff
@@ -80,42 +80,26 @@ public TimelineHolder onCreateViewHolder(ViewGroup parent, int viewType)
 	public void onBindViewHolder(final TimelineHolder holder, int position)
 	{
 		DayActivity dayActivity = (DayActivity) getItem(position);
+		holder.getView().setTag(dayActivity);
+		setUpTimeLineViewHolderWithDayActivity(holder, dayActivity);
+	}
 
+	private void setUpTimeLineViewHolderWithDayActivity(TimelineHolder holder, DayActivity dayActivity)
+	{
 		if (dayActivity != null)
```

Comment:
Can this realistically be null? The caller shouldn't call this method when it doesn't have a day activity.

Reply:
Done. Removed the null check.